resources/css/style.css

```diff
@@ -35,6 +68,33 @@
 .fa-clipboard:hover .hinttext {
   visibility: visible;
 }
+.filter-projects-inputs {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-evenly;
+  margin-top: 1rem;
+}
+.filter-btn {
+    width: 165px;
```

Comment:
two space indents.  And please add a commit to lint .css files with the same linter we use for coala & documentation repos

Reply:
Sure! I will add add that. 😇

Reply:
I mis-understood! I thought a new commit should be appended to this PR adding CSSLintBear. I was adding that only from last 2 hours. But I read the review carefully, which says that just to create an issue.
I will create a PR for that too, as that was including some a-lot changes in `travis.yml` too.

Reply:
I edited the comment to avoid too much work.